images/README.md

```diff
@@ -0,0 +1,15 @@
+# testing/images
+
+Images used for various CI jobs for cert-manager and related projects.
+
+All images are built in Prow. There is a Prow job per image in [config/jobs/testing/test-trusted.yaml](config/jobs/testing/test-trusted.yaml).
+
+Most images are built using the scripts in [image/builder](images/builder).
+
+### kind
+
+[images/kind](images/kind) contains a script to build our own KIND image- this may be useful when needing to test against a particular version of Kubernetes for which there is no 'official' KIND image yet.
```

Comment:
suggestion: em dash
```suggestion
[images/kind](images/kind) contains a script to build our own kind image---this may be useful when needing to test against a particular version of Kubernetes for which there is no 'official' kind image yet.
```